packages/editor/src/client/base.tsx

```diff
@@ -23,6 +23,8 @@ import {
 
 import {useTranslation} from 'react-i18next'
 
+import './global.less'
```

Comment:
It's already imported in `app.tsx` which uses base.tsx .. I guess you don't need to import it again in base component